app/resources/inventory.py

```diff
@@ -6,6 +6,16 @@
 from vars import game_info
 
 
+@m.user_endpoint(path=["inventory", "trade"], requires=trade_requirements)
+def trade(data: dict, user: str) -> dict:
+    element: Optional[Inventory] = wrapper.session.query(Inventory).filter_by(element_uuid=data["element_uuid"]).first()
```

Comment:
Check if the user uuid is valid. Check if this user exists
```
if not m.check_user_uuid(data["target"]):
    return {"error": "user_uuid_does_not_exists"}
```

Reply:
Yeah! @hugohabicht01 We added this feature just a few days after your task. So add this lines to check if the target is an real existing user.